### PR TITLE
Make the manifest pass installability criteria in Chrome

### PR DESCRIPTION
**Pull request by tomayac**

### CODE REVIEW

**Comment by tomayac** [comment]
Seems like @Jaifroid might be able to review this.

**Comment by tomayac** [comment]
It currently fails Chrome's installability criteria:

Screen Shot 2022-05-12 at 12 09 03

As the author of [Tabbed Application Mode](https://web.dev/tabbed-applicaiton-mode/) I know it's weird, but currently you need to pass `"display_override": ["standalaone", "tabbed"]`, which effectively disables tabbed application mode. It's still behind a flag anyways.

**Comment by Jaifroid** [comment]
OK, thanks! I'll test and report back, but in principle this PR looks great, thanks.

**Comment by kelson42** [comment]
@Jaifroid This is a side remark, but I have difficulties to deal with the milestone "High priority". I believe to understand that this means: these tickets have to be fixed **and released** quickly... but then I would propose (this is how we do in other repositories) to create a new milestone (or change current milestone) to reflect this and prepare a quick minor/patch release. Doing like this would IMO bring more clarity.

**Comment by Jaifroid** [comment]
> @Jaifroid This is a side remark, but I have difficulties to deal with the milestone "High priority". I believe to understand that this means: these tickets have to be fixed **and released** quickly... but then I would propose (this is how we do in other repositories) to create a new milestone (or change current milestone) to reflect this and prepare a quick minor/patch release. Doing like this would IMO bring more clarity.

OK, @kelson42  what would you suggest I call this milestone? I can certainly release an updated PWA once this is tested and merged to master.

**Comment by kelson42** [comment]
@Jaifroid Either `1.9.10` (a patch release, only with small bug fixes) or `1.10.0` (a minor release)

**Comment by Jaifroid** [comment]
@tomayac I'm seeing this in Chromium (Edge). Can we ignore? The value is `"false"` (in quotation marks).

image

File: manifest.json [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
   "start_url": ".",
   "short_name": "Kiwix PWA",
   "theme_color": "black",
   "description": "Offline Wikipedia reader",
   "orientation": "any",
   "background_color": "transparent",
-  "relatedapplications": "",
-  "preferrelated_applications": "false",
+  "related_applications": [],
+  "prefer_related_applications": "false",
```
Comment (tomayac) [inline manifest.json:14]:
```suggestion
  "prefer_related_applications": false,
```

**Comment by tomayac** [comment]
Oops, overlooked this. It should be boolean. I’ve also deleted the `"generated"` line, which isn’t a defined field.

**Review by Jaifroid — APPROVED** [review]
Tested and installing correctly on Chrome and Edge (Chromium).

**Comment by tomayac** [comment]
Super, this is ready to squash-merge. Thanks for the quick turn-around time!

**Comment by Jaifroid** [comment]
@tomayac Just to let you know that pwa.kiwix.org now has the new release, v1.9.93.

**Comment by tomayac** [comment]
Woohoo! Confirmed working: 
Screen Shot 2022-05-12 at 18 01 35

**Pull request merged**